surveys/2024-04/questions.md

```diff
@@ -0,0 +1,171 @@
+# Survey April 2024
+
+## Demographics
+
+- Where are you from? (countries list if possible - or open question)
+    - country
+
+- How long have you been working with Node.js? (numbers list from 0 to 10+)
+
+- In what kind of organizations are you working in?
+    - big tech
+    - company
+    - startup
+    - university
+    - individual
+    - other
+
+- Are you part/following some Working Groups? Are you involved in some Node.js community Working Groups? If yes which one (open question)
```

Comment:
I think not many people know about working groups, we might add a link to explain what it is

Reply:
Honestly I think I'd just remove that from the survey. Few people knew what we were asking and in terms of keeping the survey shorter it may be better to just remove this one.

Reply:
agree on removing.